react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component, Fragment } from 'react';
+import Spinner from 'react-spinkit';
+
+import api from '~services/MatchesService';
+
+import styles from './styles.module.scss';
+
+class Matches extends Component {
+  state = {
+    matches: [],
+    loading: true
+  };
+
+  componentDidMount() {
+    setTimeout(() => {
```

Comment:
Why this timeout?

Reply:
this timeout simulates a slow response from the api. 😅😅